corehq/apps/users/views/mobile/users.py

```diff
@@ -1123,25 +1124,44 @@ def page_url(self):
         return reverse(self.urlname, args=self.args, kwargs=self.kwargs)
 
 
-@method_decorator([FILTERED_BULK_USER_DOWNLOAD.required_decorator()], name='dispatch')
 class FilteredUserDownload(BaseManageCommCareUserView):
-    urlname = 'filter_and_download_commcare_users'
-    page_title = ugettext_noop('Filter and Download')
+    page_title = ugettext_noop('Filter and Download Users')
 
-    @method_decorator(require_can_edit_commcare_users)
     def get(self, request, domain, *args, **kwargs):
-        form = CommCareUserFilterForm(request.GET, domain=domain, couch_user=request.couch_user)
+        form = UserFilterForm(request.GET, domain=domain, couch_user=request.couch_user, user_type=self.user_type)
         # To avoid errors on first page load
         form.empty_permitted = True
         context = self.main_context
-        context.update({'form': form, 'count_users_url': reverse('count_users', args=[domain])})
+        context.update({'form': form, 'count_users_url': reverse(self.count_view, args=[domain])})
         return render(
             request,
             "users/filter_and_download.html",
             context
         )
 
 
+@method_decorator([require_can_use_filtered_user_download], name='dispatch')
+class FilteredCommCareUserDownload(FilteredUserDownload):
```

Comment:
Since this PR is making it cleaner user types by using mobile and web, instead of using "commcare" for mobile, do we want to name this as `FilteredMobileUserDownload` ?

Reply:
I see what you're saying, but I'm going to call this out of scope. The code is a mix of "commcare user" and "mobile user", I'm not sure which one wins out.